### DISCUSSION TITLE
페이지 레이아웃에 대한 아이디어 1

### DISCUSSION THREAD

**Discussion by sosorini** [sulrung-boardgame/set-front#13] [Ideas]
- 모바일이고 한 페이지에서 다 보여줘야 하니 헤더나 푸터에 나포함 방멤버들의 점수를 간략하게 보여줘야 하나? 싶은데 어떻게 생각하나여

**Comment by JennyLee4517** [comment]
와우 github에 이런기능 있는거 처음봐요.. 싱기방기 +_+
방멤버들의 점수 보여주는거 찬성이요~

**Reply by sosorini** [reply]
ㅋㅋㅋ #7 에서 작업해보니 width가 320px 일 때 카드 보여주기도 벅찬? UI 가 되는 것 같아서
간단하게 나타내면 좋을 거 같아용
이름 다 보여주고 점수 보여주기엔 좀 좁을 것 같기도? 해서 아이디어가 필요합니당
<img width="337" alt="스크린샷 2023-06-05 오후 1 52 34" src="https://github.com/sulrung-boardgame/set-front/assets/45188497/b3f049e0-697d-4534-a94b-35b1539daf93">

**Reply by JennyLee4517** [reply]
근데 사실 실제로 오프라인에서 게임하다보면 남들 몇점땄는지 볼 겨를이 없긴해요 ㅋㅋㅋㅋ 카드 찾기두 바빠서...
아예 빼도 괜찮을것 같습니다??
